breacharbiter.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/lightningnetwork/lnd/htlcswitch"
 	"github.com/lightningnetwork/lnd/input"
 	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/lnwallet/chainfees"
```

Comment:
typically go recommends singular package names, which we abide by for the most part. "chain fee estimator"  i think sounds better than "chain fees estimator", the same holds for many of the other types